docs/reference/examples.md

```diff
@@ -2,150 +2,95 @@
+The `js-soroban-client` exposes the [`TransactionBuilder`](https://stellar.github.io/js-stellar-base/TransactionBuilder.html) class from `js-stellar-base`.  There are more examples of [building transactions here](https://github.com/stellar/js-stellar-base/blob/master/docs/reference/base-examples.md). All those examples can be signed and submitted to Stellar in a similar manner as is done below.
 
 In this example, the destination account must exist. The example is written
 using modern Javascript, but `await` calls can also be rendered with promises.
 
 ```javascript
-// Create, sign, and submit a transaction using JS Stellar SDK.
+// Create, sign, and submit a transaction using JS Soroban Client.
 
 // Assumes that you have the following items:
 // 1. Secret key of a funded account to be the source account
 // 2. Public key of an existing account as a recipient
 //    These two keys can be created and funded by the friendbot at
 //    https://laboratory.stellar.org under the heading "Quick Start: Test Account"
-// 3. Access to JS Stellar SDK (https://github.com/stellar/js-stellar-sdk)
+// 3. Access to JS Soroban Client (https://github.com/stellar/js-soroban-client)
 //    either through Node.js or in the browser.
 
-// This code can be run in the browser at https://laboratory.stellar.org
-// That site exposes a global StellarSdk object you can use.
-// To run this code in the Chrome, open the console tab in the DevTools.
-// The hotkey to open the DevTools console is Ctrl+Shift+J or (Cmd+Opt+J on Mac).
-const StellarSdk = require('stellar-sdk');
+const SorobanClient = require('soroban-client');
 
 // The source account is the account we will be signing and sending from.
 const sourceSecretKey = 'SCZANGBA5YHTNYVVV4C3U252E2B6P6F5T3U6MM63WBSBZATAQI3EBTQ4';
 
 // Derive Keypair object and public key (that starts with a G) from the secret
-const sourceKeypair = StellarSdk.Keypair.fromSecret(sourceSecretKey);
+const sourceKeypair = SorobanClient.Keypair.fromSecret(sourceSecretKey);
 const sourcePublicKey = sourceKeypair.publicKey();
 
 const receiverPublicKey = 'GAIRISXKPLOWZBMFRPU5XRGUUX3VMA3ZEWKBM5MSNRU3CHV6P4PYZ74D';
 
-// Configure StellarSdk to talk to the horizon instance hosted by Stellar.org
+const contractId = '0000000000000000000000000000000000000000000000000000000000000001';
+
+// Configure SorobanClient to talk to the horizon instance hosted by Stellar.org
 // To use the live network, set the hostname to 'horizon.stellar.org'
-const server = new StellarSdk.Server('https://horizon-testnet.stellar.org');
+const server = new SorobanClient.Server('https://soroban-rpc-testnet.stellar.org');
```

Comment:
sanity check: is this url correct?

Reply:
It doesn't exist, no. But there's no public one, so I wasn't sure where to set it to tbh. Maybe we should just use localhost for now... 🤔